src/ontology/mondo-edit.obo

```diff
@@ -198459,14 +198459,14 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0009681
-name: Ullrich congenital muscular dystrophy 1
+name: Ullrich congenital muscular dystrophy 1a
```

Comment:
1A should be capital:
https://omim.org/entry/254090

Reply:
done